File: get-feature-15.js
const sqlite3 = require('sqlite3').verbose();
const path = require('path');

const dbPath = path.join(__dirname, 'features.db');
const db = new sqlite3.Database(dbPath);

db.get("SELECT * FROM features WHERE id = 15", (err, row) => {
  if (err) {
    console.error('Error:', err);
    process.exit(1);
  }

  if (row) {
    console.log(JSON.stringify(row, null, 2));
  } else {
    console.log('Feature #15 not found');
  }

  db.close();
});

File: get-feature-15.py
import sqlite3
import json
import os

db_path = os.path.join(os.path.dirname(__file__), 'features.db')
conn = sqlite3.connect(db_path)
cursor = conn.cursor()

cursor.execute("SELECT * FROM features WHERE id = 15")
row = cursor.fetchone()

if row:
    columns = [desc[0] for desc in cursor.description]
    feature = dict(zip(columns, row))
    print(json.dumps(feature, indent=2))
else:
    print('Feature #15 not found')

conn.close()
